tests/model/range.js

```diff
@@ -593,20 +593,22 @@ describe( 'Range', () => {
 			expect( transformed[ 0 ].end.path ).to.deep.equal( [ 4, 7 ] );
 		} );
 
-		it( 'should stick to moved range, if the transformed range is collapsed #1', () => {
+		it( 'should not stick to moved range, if the transformed range is collapsed #1', () => {
```

Comment:
Are you sure that is should not move, if we already have tests checking the opposite behavior?